docs/assembly.rst

```diff
@@ -104,47 +104,48 @@ efficient code, for example:
 
     pragma solidity >=0.4.16 <0.7.0;
 
+
     library VectorSum {
         // This function is less efficient because the optimizer currently fails to
         // remove the bounds checks in array access.
-        function sumSolidity(uint[] memory _data) public pure returns (uint o_sum) {
```

Comment:
Why not just call it `sum`?

Reply:
Seems fine to me! I have no idea who named it `o_sum` or `oSum` and if there was a reason for it, but it seems reasonable var name to me.